src/Components/Game/Game.js

```diff
@@ -103,23 +103,19 @@ class Game extends React.Component {
     }
   }
 
-  handleChangeScoreClick() {
-    this.setState((prevState) => {
```

Comment:
The problem of the non-working score was caused by mutating score: `prevState.score++` and `prevState.pointMultiplier++`. The state should never be mutated